dashboard/src/components/Sheet/LogOrJsonSheetContent.tsx

```diff
@@ -1,44 +1,58 @@
 import type { LinkProps } from '@tanstack/react-router';
 
+import ReactJsonView from '@microlink/react-json-view';
+
 import type { TNavigationLogActions } from '@/components/Sheet/WrapperSheetContent';
 import { WrapperSheetContent } from '@/components/Sheet/WrapperSheetContent';
 
 import { MemoizedMoreDetailsButton } from '@/components/Button/MoreDetailsButton';
 
-import { LogViewerCard } from './LogViewerCard';
-import { LogExcerpt } from './LogExcerpt';
+import { LogViewerCard } from '@/components/Log/LogViewerCard';
+import { LogExcerpt } from '@/components/Log/LogExcerpt';
+
+export type SheetType = 'log' | 'json';
 
 interface ILogSheet {
+  type?: SheetType;
+  jsonContent?: JSON;
   logExcerpt?: string;
   logUrl?: string;
   navigationLogsActions?: TNavigationLogActions;
   currentLinkProps?: LinkProps;
 }
 
-export const LogSheetContent = ({
+export const LogOrJsonSheetContent = ({
+  type = 'log',
+  jsonContent,
   logExcerpt,
   logUrl,
   navigationLogsActions,
   currentLinkProps,
 }: ILogSheet): JSX.Element => {
   return (
     <WrapperSheetContent
-      sheetTitle="logSheet.title"
+      sheetTitle={type === 'log' ? 'logSheet.title' : 'jsonSheet.title'}
       navigationLogsActions={navigationLogsActions}
       detailsButton={
         currentLinkProps && (
           <MemoizedMoreDetailsButton linkProps={currentLinkProps} />
         )
       }
     >
-      <LogViewerCard
-        logUrl={logUrl}
-        isLoading={navigationLogsActions?.isLoading}
-      />
-      <LogExcerpt
-        logExcerpt={logExcerpt}
-        isLoading={navigationLogsActions?.isLoading}
-      />
+      {type === 'log' ? (
+        <>
+          <LogViewerCard
+            logUrl={logUrl}
+            isLoading={navigationLogsActions?.isLoading}
+          />
+          <LogExcerpt
+            logExcerpt={logExcerpt}
+            isLoading={navigationLogsActions?.isLoading}
+          />
+        </>
+      ) : (
+        <ReactJsonView src={jsonContent ?? {}} />
```

Comment:
```suggestion
        <ReactJsonView
          src={jsonContent ?? {}}
          style={{ fontSize: 14 }}
          displayDataTypes={false}
          name={false}
        />
```
The font size is smaller than usual, I don't think it's necessary to show the datatypes and you don't need the "root" at the beginning (it would be interesting if it showed the name of the json object you are passing to the viewer, like "logspec", but this would require other changes)